### PR TITLE
Describe serialization config location in cookbook

### PR DESCRIPTION
**Pull request by wouterj**
Finishes https://github.com/symfony/symfony-docs/pull/6039

Original description:

> Text from : http://symfony.com/blog/new-in-symfony-2-7-serializer-component-improvements

### CODE REVIEW

File: cookbook/serializer.rst [hunk 1]
```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
         $someObject,
         'json', array('groups' => array('group1'))
     );
+    
+
```
Comment (xabbuh) [inline cookbook/serializer.rst]: One blank line should be removed.

File: cookbook/serializer.rst [hunk 2]
```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
...
+Instead of ``@Groups`` annotations, the Serializer component also supports Yaml
```
Comment (xabbuh) [inline cookbook/serializer.rst]: In addition to the `@Groups` annotation, [...]

File: cookbook/serializer.rst [hunk 3]
```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
...
+or XML files. These configuration are automatically loaded from these locations:
```
Comment (xabbuh) [inline cookbook/serializer.rst]: These files are automatically when being stored in one of the following locations:

File: cookbook/serializer.rst [hunk 4]
```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
...
+
+ * The ``serialization.yml`` or ``serialization.xml`` file in
+   ``Resources/config/`` of a bundle;
+ * All ``*.yml`` and ``*.xml`` files in ``Resources/config/serialization/`` of
```
Comment (xabbuh) [inline cookbook/serializer.rst]: [...] in the `Resources/config/serialization/` directory [...]

File: cookbook/serializer.rst [hunk 5]
```diff
@@ -171,6 +171,15 @@ to your class and choose which groups to use when serializing::
         $someObject,
         'json', array('groups' => array('group1'))
     );
+    
+
+Instead of ``@Groups`` annotations, the Serializer component also supports Yaml
+or XML files. These configuration are automatically loaded from these locations:
+
+ * The ``serialization.yml`` or ``serialization.xml`` file in
```
Comment (xabbuh) [inline cookbook/serializer.rst]: [...] in the `Resources/config/` directory [...]

**Comment by javiereguiluz** [comment]
👍 after making the changes proposed by @xabbuh.

Status: reviewed

**Pull request merged**